src/runtime/math.py

```diff
@@ -312,8 +317,18 @@ def trunc(x: f32) -> i32:
         return ceil(x)
 
 def sqrt(x: f64) -> f64:
+    """
+    Returns square root of a non negative x
+    """
+
     return x**(1/2)
 
+def cbrt(x: f64) -> f64:
+    """
+    Returns cube root of a number x
+    """
+    return x**(1/3)
```

Comment:
Do these expressions actually work with Lpython?

Reply:
yeah, they should.